test/WebSites/LocalizationWebSite/Views/Shared/_LocalizationLayout.cshtml

```diff
@@ -0,0 +1,9 @@
+﻿@using System.Globalization
+@using Microsoft.Framework.Localization
+@using Microsoft.AspNet.Mvc.Localization
+@inject IStringLocalizerFactory StringLocalizerFactory
+@{
+    var locLayoutString = StringLocalizerFactory.Create("LocalizationWebSite.Views.Shared._LocalizationLayout.cshtml", "");
```

Comment:
`ViewLocalizer` does not read resource string from layouts since `ViewContext` does not show `layout` as a view. I will file a bug as we talked to address this issue.